### DISCUSSION TITLE
Anyone facing keyboard shortcuts randomly don't work in code-server 4.20.0 or vscode 1.85.1?

### DISCUSSION THREAD

**Discussion by bilogic** [coder/code-server#6601] [Q&A]
I customized `F6` to `Reveal file in explorer`, so the sidebar opens automatically, switches to explorer and reveals the file. When the issue occures in, `4.20.0`, nothing happens when I press `F6`, other times it works.

I noticed this issue happening on my W10 Chrome `Version 120.0.6099.130 (Official Build) (64-bit)` in `4.20.0`, so reverting to `4.19.1-rc.2` for now.

~My question is, has anyone faced a similar problem?~

UPDATE1: I switch several times between 4.19.1-rc.2 and pretty certain this problem is exists `4.20.0` and not in `4.19.1-rc2` (other versions untested), just can't be sure if it was due to 1.85.1, need time or could use some help to reproduce the problem consistently.

UPDATE2: did a quick test but unable to reproduce on vscode.dev (Version: 1.85.1 Commit: 0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2)

**Comment by code-asher** [comment]
Could it be a conflict with one of the other F6 keybindings?  I tried binding `workbench.files.action.showActiveFileInExplorer` to `F6` and so far it seems to be working for me in v4.20.0 although I did not do much, just opened a couple files and hit F6.  Also experimented with closing the sidebar and it opens as expected.

I am using Chromium 120.0.6099.129 on Linux.

**Reply by bilogic** [reply]
Thanks. My observations indicate it is from vscode https://github.com/microsoft/vscode/issues/201637
